plugins/push/frontend/public/templates/push-notification-details.html

```diff
@@ -1,3 +1,93 @@
 <script type="text/x-template" id="push-notification-details">
-    <h2>Placeholder for push notification details page </h2>
+    <div v-bind:class="[componentId]">
+        <cly-header class="cly-vue-push-notification-details-header">
+            <template v-slot:header-left>
+                <h2> {{pushNotification.message.name }} </h2>
+            </template>
+            <template v-slot:header-top>
+                <cly-back-link link="/messaging" :title="i18n('push-notification-details.back-to')"></cly-back-link>
+            </template>
+            <template v-slot:header-right>
+                <cly-more-options size="small" @command="handleUserEvents($event, scope.row._id )">
+                    <el-dropdown-item command="UserEventEnum.DUPLICATE">{{i18n('push-notification.duplicate')}}</el-dropdown-item>
+                    <el-dropdown-item command="UserEventEnum.DELETE">{{i18n('push-notification.delete')}}</el-dropdown-item>
+                    <el-dropdown-item command="UserEventEnum.RESEND">{{i18n('push-notification.resend')}}</el-dropdown-item>
```

Comment:
If you need values of these enums, you should have used binding. For instance ":command=UserEventEnum.DUPLICATE". But I guess you need literals themselves rather than values.

Reply:
Actually I need the values but since the actions are not specified yet for the push plugin, I have not tested them yet. I will have to add the vue bindings. Good catch!